.github/workflows/chart-release-on-push-to-branch.yml

```diff
@@ -0,0 +1,25 @@
+name: Push Charts to helm-charts-registry on push to branch
+
+on:
+  push:
+    branches-ignore:
+      - 'main'
+    paths:
+      - 'charts/**'
```

Comment:
if changes are done that are only application wide, no helm chart gets published for the branch. In this case the pipeline would have to use a default (main) Helm Chart. This should be talked about in the Topic Team and taken into consideration